Microsoft.Azure.Cosmos/src/Routing/AvailabilityStrategy/AvailabilityStrategy.cs

```diff
@@ -19,23 +16,26 @@ namespace Microsoft.Azure.Cosmos
     abstract class AvailabilityStrategy
     {
         /// <summary>
-        /// Execute the availability strategy
+        ///  stuff
         /// </summary>
-        /// <param name="sender"></param>
-        /// <param name="client"></param>
-        /// <param name="requestMessage"></param>
-        /// <param name="cancellationToken"></param>
-        /// <returns>The response from the service after the availability strategy is executed</returns>
-        internal abstract Task<ResponseMessage> ExecuteAvailabilityStrategyAsync(
-            Func<RequestMessage, CancellationToken, Task<ResponseMessage>> sender,
-            CosmosClient client,
-            RequestMessage requestMessage,
-            CancellationToken cancellationToken);
+        /// <returns>something</returns>
+        public static AvailabilityStrategy DisabledAvailabilityStrategy()
```

Comment:
Regarding method name, repeating AvailabilityStrategy when the class is already AvailabilityStrategy might be repetitive?

What about
```suggestion
        public static AvailabilityStrategy DisabledStrategy()
```

@kirankumarkolli thoughts?